pytest_tests/testsuites/failovers/test_failover_network.py

```diff
@@ -110,3 +114,56 @@ def test_block_storage_node_traffic(
                 wallet, cid, oid, shell=self.shell, endpoint=new_nodes[0].get_rpc_endpoint()
             )
             assert get_file_hash(source_file_path) == get_file_hash(got_file_path)
+
+    @pytest.mark.sanity
+    @allure.title("RPC reconnection test")
+    def test_rpc_reconnection(self, hosting: Hosting):
+        """
+        When RPC connection fails (and it can), storage node reconnects to some other node and continues to operate.
+        """
+        dport_repeat = 50  # Constant for the number of the disconnect should be repeated
+        morph_chain_endpoints = get_morph_chain_endpoints(hosting)
+
+        for storage_node in hosting.find_service_configs(STORAGE_NODE_SERVICE_NAME_REGEX):
+            host = hosting.get_host_by_service(storage_node.name)
+            pid = host.get_service_pid(storage_node.name)
+
+            for morph_chain_addr, morph_chain_port in morph_chain_endpoints:
+                with allure.step(f'Disconnecting storage node {storage_node.name} '
+                                 f'from {morph_chain_addr} {dport_repeat} times'):
+                    for _ in range(dport_repeat):
+                        try:
+                            """
+                            Of course, it would be cleaner to use such code:
+                            with Namespace(pid, 'net'):
+                                subprocess.check_output(['ss', '-K', 'dst', addr, 'dport', port])
+                            But it would be required to run the tests from root, which is bad practice.
+                            But we face the limitations of the ubuntu-latest runner:
+                            And using setfacl is not possible due to GitHub ubuntu-latest runner limitations.
+                            """
+                            command = f'ss -K dst {morph_chain_addr} dport {morph_chain_port}'
+                            sudo_command = f'sudo nsenter -t {pid} -n {command}'
+                            output = subprocess.check_output(sudo_command, shell=True)
+                            logger.info(f'Output of the command {sudo_command}: {output}')
+                        except subprocess.CalledProcessError as e:
+                            logger.error(
+                                f'Error occurred while running command: {sudo_command}. Error message: {str(e)}')
+                            raise
+                    logger.info(
+                        f'Disconnected storage node {storage_node.name} from {morph_chain_addr} {dport_repeat} times')
+
+            for node in self.cluster.storage_nodes:
+                with allure.step(f'Checking if node {node} is alive'):
+                    try:
+                        health_check = storage_node_healthcheck(node)
+                        assert (
+                                health_check.health_status == "READY"
+                                and health_check.network_status == "ONLINE"
```

Comment:
I'm not sure it's a sufficient check. I'd try getting a network map from it.

Reply:
> I'm not sure it's a sufficient check. I'd try getting a network map from it.

I've added a node check in the map

Reply:
I mean something like `neofs-cli netmap netinfo`. Node can't respond to this request successfully if it's not connected to blockchain RPC node.

Reply:
> I mean something like `neofs-cli netmap netinfo`. Node can't respond to this request successfully if it's not connected to blockchain RPC node.

Done